utils/gitlab_api.py

```diff
@@ -188,6 +188,46 @@ def create_app_interface_reporter_mr(self, reports, email_schema,
         return self.create_mr(branch_name, target_branch,
                               commit_message, labels=labels)
 
+    def create_app_interface_notificator_mr(
+        self, notification,
+        notification_path="/app-interface",
+        email_schema="/app-interface/app-interface-email-1.yml"
+    ):
+        # labels = ['automerge']
+        labels = ['do-not-merge/hold']
+        prefix = 'app-interface-notificator'
+        target_branch = 'master'
+
+        now = datetime.now()
+        ts = now.strftime("%Y%m%d%H%M%S")
```

Comment:
Good point! I'll include notification["short_description"] in branch_name in addition to datatime info.

Reply:
the short description may be too long for the branch name, it may also contain spaces or characters that are not allowed in branch names.

I guess the branch name could be something like `notificator-<unixtime>` where `<unixtime>` is the result of `int(time.time())`. That would make it short and unique

Reply:
I see. I didn't think about that description may contain spaces/ unallowed chars. I will change the branch name to `notificator-<unixtime>`. Thank you, JF!